src/main/webapp/WEB-INF/aps/jsp/models/home.jsp

```diff
@@ -15,43 +15,37 @@
-            <div class="spacer-login-lp"></div>
-
-            <div class="entando-intro">
-                If you can see this page, then the people who own this host have just installed <strong>Entando</strong> successfully.<br/>
-                They now have to replace this placeholder page or just its contents.
-            </div>
-            <div class="spacer-landing"></div>
-            <div class="entando-intro">
-                You can go to the login form and use the following credentials to Enter the Administration Area
-            </div>
+        <div class="container e6-home-container">
+            <div>
+                <div class="logo-entando-top">
+                    <img class="logo-entando-login" src="<wp:resourceURL />administration/img/login-logo.svg" />
+                </div>
+                <div class="ux_brand-lp">The Entando <wp:info key="systemParam" paramName="version" /> has been successfully installed</div>
+                <div class="entando-intro lgfont">
+                    This is the default page after installing Entando successfully.
+                </div>
+                <div class="entando-intro">
+                    To replace this page, go to the home page and login as the administrator.
+                </div>
 
-            <div class="entando-intro">
-                Username: <strong>admin</strong> <br /> Password: <strong>adminadmin</strong>
-            </div>
-
-            <div class="login-buttons-lp">
+                <div class="entando-intro">
+                    <p><strong>Default Username:</strong> admin</p>
+                    <p><strong>Default Password:</strong> adminadmin</p>
+                </div>
+                
                 <a href="<jstl:url value="/do/login"/>" class="btn btn-login">
                     GO TO LOG IN PAGE
                 </a>
-            </div>
 
-            <div id="social-link">
-                <a href="http://www.entando.com" target="_blank" rel="noopener noreferrer"><img src="<wp:resourceURL />administration/img/entando_icon.png"></a>
-                <a href="http://github.com/entando/" target="_blank" rel="noopener noreferrer"><img src="<wp:resourceURL />administration/img/github.png"></a>
-                <a href="https://twitter.com/entando" target="_blank" rel="noopener noreferrer"><img src="<wp:resourceURL />administration/img/twitter.png"></a>
+                <div id="social-link">
+                    <a href="http://www.entando.com" target="_blank"><img src="<wp:resourceURL />administration/img/entando_icon.png"></a>
+                    <a href="http://github.com/entando/" target="_blank"><img src="<wp:resourceURL />administration/img/github.png"></a>
+                    <a href="https://twitter.com/entando" target="_blank"><img src="<wp:resourceURL />administration/img/twitter.png"></a>
+                </div>
+                <div class="copyright-entando">Copyright 2020 <span class="entando-sm-write">Entando</span></div>

```

Comment:
Please generate the year so we don't have to change it every time

Reply:
I tried to use `fmt` lib for the year (same as the one on login) but it didn't work. I'm not sure why. So I made a workaround using `SimpleDateFormat` instead.